src/librbd/cache/pwl/AbstractWriteLog.h

```diff
@@ -33,6 +34,7 @@ class SyncPointLogEntry;
 class GenericWriteLogEntry;
 class WriteLogEntry;
 class GenericLogEntry;
+struct WriteLogPmemEntry;
```

Comment:
Nit: while here, can we rearrange these in alphabetical order?

Reply:
Done